webapp-django/crashstats/documentation/static/documentation/css/documentation.less

```diff
@@ -153,6 +153,58 @@
             border: none;
             margin: 0;
         }
+
+        button.try {
+            background-color: green;
+            background-image: none;
+            border: none;
+            box-shadow: none;
+            float: right;
+            font-size: 0.9em;
+            font-weight: bold;
+            padding: 0 2px;
+            margin: 0;
+            min-width: 0;
+        }
+
+        .results {
+            background-color: black;
+            border-top: 1px solid #999;
+
+            .loader {
+                background-image: url('../../img/ajax-loader16x16.gif');
```

Comment:
This is the loader gif we're supposed to use for things with a white background. That's why it looks a bit odd when it's over the soon-to-be-expanded Results panel thing.